dev/io.openliberty.jakarta.batch.2.0/pom.xml

```diff
@@ -0,0 +1,134 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>jakarta.batch</groupId>
+        <artifactId>jbatch</artifactId>
+        <version>2.0</version>
+    </parent>
+    <groupId>jakarta.batch</groupId>
+    <artifactId>jakarta.batch-api</artifactId>
+    <packaging>jar</packaging>
+    <version>2.0.0</version>
+    
+    <properties>
+    	<non.final>false</non.final>
+    	<extension.name>jakarta.batch</extension.name>
+    	<spec.version>2.0.0</spec.version>
+    </properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>jakarta.inject</groupId>
```

Comment:
Is this accurate?  I don't have group jakarta.inject with artifact jakarta.inject version 2 in maven.  Should it be group jakarta.inject with artifact jakarta.inject-api with a version of 2.0.0?